packages/lumx-react/src/components/link-preview/LinkPreview.tsx

```diff
@@ -95,35 +111,40 @@ export const LinkPreview: Comp<LinkPreviewProps, HTMLDivElement> = forwardRef((p
 
                 <div className={`${CLASSNAME}__container`}>
                     {title && (
-                        <div className={`${CLASSNAME}__title`}>
+                        <h2 className={`${CLASSNAME}__title`}>
```

Comment:
I'm not too sure about forcing this to be a h2.
As this is a highly reusable component, it would break the hiearchy to have it beneath an h3 for example.

Could we have a prop to choose the title level?